com.github.limads.Queries.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id" : "com.github.limads.Queries",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "queries",
+    "finish-args" : [
+        "--share=network",
+        "--device=dri",
+        "--socket=wayland",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--filesystem=home"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+	        "name" : "Queries",
+	        "builddir" : true,
+	        "buildsystem" : "simple",
+	        "build-commands" : [
+	            "sh ${FLATPAK_BUILDER_BUILDDIR}/scripts/build.sh"
+	        ],
+	        "sources" : [
+	        
+	        {
+	            "type" : "archive",
+	            "url" : "https://limads.sfo3.cdn.digitaloceanspaces.com/queries-vendored/v0.1/vendored.tar.gz",
+	            "sha256" : "7add2c5672450ad4e48fe1d84d921524260a2c0c27c826f6dc6b6727787410d8"
+	        },
+	        
+		    {
+		        "type" : "git",
+		        "url" : "https://github.com/limads/queries.git"
+		    }
+		    
+	        ],
+	        "post-install" : [
+	        	"sh ${FLATPAK_BUILDER_BUILDDIR}/scripts/postinstall.sh"
```

Comment:
I had a look at this script and it does many things wrong.

Reply:
Thanks for taking a look. I've just examined the script and failed to see the error - The executable and the symbolic/application icons are all found after the installation.

Reply:
1. it's all specific to flatpak becaus it hardcode a prefix.
2. it should never put anything in `share/app-info/`. Not sure what piece of documentation told you to install anything in there.
3. it needlessly use mkdir when install handle creating directories.
4. icons arent even in the right place
5. this should either be in a proper build system (cargo is too limited) or put as individual commands in the manifest here.